editor/plugins/canvas_item_editor_plugin.cpp

```diff
@@ -3645,7 +3645,7 @@ void CanvasItemEditor::_draw_hover() {
 }
 
 void CanvasItemEditor::_draw_transform_message() {
-	if (drag_selection.is_empty() || !drag_selection.front()->get()) {
+	if (drag_type == DRAG_NONE || drag_selection.is_empty() || !drag_selection.front()->get()) {
```

Comment:
When the scene tab is switched, `drag_selection` contains nodes that are not in the tree. 
```suggestion
	if (drag_selection.is_empty() || !drag_selection.front()->get() || !drag_selection.front()->get()->is_inside_tree() ) {
```

Reply:
But `drag_type` is still `DRAG_NONE`, so contents of `drag_selection`.

I agree that clearing the selection would be better solution, but it's not obvious when it should be done.